components/crowdfunding-redesign/Fundraiser.tsx

```diff
@@ -190,46 +195,51 @@ const ContentOverview = ({ content }) => {
         About
       </Link>
       {headingTexts.map(heading => (
-        <Link href="#" key={heading} className={linkClasses()}>
+        <Link href="#" key={heading} className={linkClasses()} onClick={triggerPrototypeToast}>
           {heading}
         </Link>
       ))}
     </div>
   );
 };
 
-export default function Fundraiser(props) {
-  const account = props.account;
-
-  const title = account.parent ? account.name : `Support ${account.name}`;
+export default function Fundraiser({ account }) {
+  const fundraiser = getDefaultFundraiserValues(account);
   const mainAccount = account.parent || account;
   return (
     <React.Fragment>
       <div className="border-b bg-background">
-        <div className="mx-auto flex h-16 max-w-screen-xl items-center justify-between">
+        <div className="mx-auto flex h-16 max-w-screen-xl items-center justify-between px-6">
           <Link href={`/preview/${account.parent?.slug || account.slug}`}>
             <Avatar className="" collective={account.parent || account} />
           </Link>
         </div>
       </div>
       <div className="">
-        <div className="mx-auto max-w-screen-xl space-y-8 py-12">
+        <div className="mx-auto max-w-screen-xl space-y-8 px-6 py-12">
           <div className="space-y-4 text-center">
-            <h1 className="text-balance text-4xl font-semibold">{title}</h1>
-            <p className="text-lg">{account.description}</p>
+            <h1 className="text-balance text-4xl font-semibold">{fundraiser.name}</h1>
+            <p className="text-lg">{fundraiser.description}</p>
           </div>
 
           <div className="flex grow gap-8">
             {/* main */}
             <div className="w-full max-w-[900px] grow space-y-4">
-              <div className="relative h-96 w-full overflow-hidden rounded-lg bg-primary/20">
-                {account.backgroundImageUrl && (
-                  <img src={account.backgroundImageUrl} alt="background" className="h-full w-full object-cover" />
-                )}
+              <div className="relative aspect-video w-full overflow-hidden rounded-lg bg-primary/20">
+                {fundraiser.cover?.type === 'IMAGE' ? (
+                  // eslint-disable-next-line jsx-a11y/alt-text
+                  <img src={fundraiser.cover.url} className="h-full w-full object-cover" />
+                ) : fundraiser.cover?.type === 'VIDEO' ? (
+                  <div className="h-full w-full">
+                    <LiteYouTubeEmbed
+                      id={getYouTubeIDFromUrl(fundraiser.cover.videoUrl)}
+                      title={`Cover Video for ${fundraiser.name}`}
+                    />
```

Comment:
Any reason to use that rather than `components/VideoPlayer.tsx`? The later support YouTube + Vimeo, and uses a privacy-improved version of YT

Reply:
This package will improve loading speed by not loading the iframe and associated javascript before the user clicks on the video to play it, and instead using an image and css to display the preview. More info here: https://github.com/paulirish/lite-youtube-embed (using a [React port of this library](https://github.com/ibrahimcesar/react-lite-youtube-embed), which is also using the "nocookies" version of Youtube by default)

Reply:
@Betree But makes sense to use `VideoPlayer.tsx`, and rather update that to use this library if you think it sounds like a nice improvement? There are similar things for Vimeo

Reply:
I missed that this library was doing that, it's a good improvement indeed (also for privacy reasons) that would be great to port to other places. Updating `VideoPlayer` to that library sounds like a good solution, but we can do it separately if we're concerned about breaking changes.